x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -960,6 +960,9 @@ processors:
  - lowercase:
      field: "file.type"
      ignore_failure: true
+ - lowercase:
+     field: "network.direction"
```

Comment:
@adriansr Make sure the possible values are actually one of those listed for `network.direction` in ECS https://www.elastic.co/guide/en/ecs/current/ecs-network.html

E.g. If Cisco has "Inbound", then just lowercasing that is fine. But if instead of "unknown" they have "Unavailable", then just the lowercasing doesn't get you there completely :-)

Reply:
Yes, in these logs the direction is not a variable but a hardcoded part of the message, like "Deny inbound {protocol} for {ip}", and then there is another similar message with a different code for "Deny outbound {protocol} for {ip}". Just with message 106001, the direction was at the beginning of the message so it started with an uppercase character.